rtgui/cropwindow.cc

```diff
@@ -66,20 +66,19 @@ ZoomStep zoomSteps[] = {
 #define ZOOM11INDEX  13
 
 CropWindow::CropWindow (ImageArea* parent, rtengine::StagedImageProcessor* ipc_, bool isLowUpdatePriority_, bool isDetailWindow)
-    : ObjectMOBuffer(parent), onResizeArea(false), deleted(false), fitZoomEnabled(true), fitZoom(false), isLowUpdatePriority(isLowUpdatePriority_),
-      backColor(options.bgcolor), decorated(true), titleHeight(30),
-      sideBorderWidth(3), lowerBorderWidth(3), upperBorderWidth(1), sepWidth(2),
-      xpos(30), ypos(30), imgX(-1), imgY(-1), imgW(1), imgH(1), iarea(parent),
-      cropZoom(0), cropgl(NULL), pmlistener(NULL), observedCropWin(NULL), ipc(ipc_), isFlawnOver(false)
+    : ObjectMOBuffer(parent), state(SNormal), press_x(0), press_y(0), action_x(0), action_y(0), pickedObject(-1), pickModifierKey(0), rot_deg(0), onResizeArea(false), deleted(false),
```

Comment:
You are right that this could work in the opposite direction as moving inline method out of the class body. But considering that you cannot hide the data members in C++ (without resorting to an indirection like a pimpl) anyway, that part of the implementation is always exposed anyways. And I would only suggest to use this to initialize simple types, not complicated objects, which would otherwise just clutter up the constructors.